solutions/EMFSolutionYAMTL/solution.ini

```diff
@@ -1,5 +1,5 @@
 [build]
-default=./gradlew clean fatJar
+default=./gradlew --refresh-dependencies clean fatJar
```

Comment:
Is the `--refresh-dependencies` option needed by other "tools", e.g. batch and expl?
Should it be run for the variant that skip tests?

Reply:
Should I put `--refresh-dependencies` option to all Gradle builds of YAMTL like [here](https://github.com/TransformationToolContest/ttc2018liveContest/pull/107#discussion_r631438170)?

Reply:
This is only needed in the first YAMTL solution that is executed to make sure the latest snapshot for the stated version is downloaded in the .m2 repository, skipping the cached dependency. The other two subsequent solutions will use the cached version.

Reply:
Fine. I'll add this also to the skipTests variant for the first solution.